src/components/molecules/ProfilePictureInput/ProfilePictureInput.tsx

```diff
@@ -76,9 +88,15 @@ const ProfilePictureInput: React.FunctionComponent<PropsType> = ({
   };
 
   const uploadDefaultAvatar = async (avatar: string) => {
-    setValue("pictureUrl", process.env.PUBLIC_URL + `/avatars/${avatar}`, true);
+    setValue("pictureUrl", avatar, true);
   };
 
+  const isLoadingAvatars = customAvatars.loading;
```

Comment:
You can extract this directly inline from the `useAsync` call:

```typescript
const { loading: isLoadingAvatars, value, etc } = useAsync(
```

Reply:
Changed in https://github.com/sparkletown/sparkle/commit/6cdbdb2d81f61c67a0665f91bf358d09fc89bfbe